src/applications/ivc-champva/10-10D/components/CustomRelationshipPattern.jsx

```diff
@@ -0,0 +1,145 @@
+import {
+  radioUI,
+  radioSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import VaTextInputField from 'platform/forms-system/src/js/web-component-fields/VaTextInputField';
+
+/**
+ * Customized web component uiSchema for relationship to veteran
+ * with support for more custom wording.
+ *
+ * set `labelHeaderLevel: ''` if you just want to use it as a standard field instead
+ *
+ * ```js
+ * relationshipToVeteran: relationshipToVeteran() // 'Veteran'
+ * relationshipAsField: relationshipToVeteran({
+ *  personTitle: 'claimant',
+ *  labelHeaderLevel: ''
+ * })
+ * customRelationshipToVeteran: relationshipToVeteranUI({
+ *   customLabels: {
+ *     caretaker: "I’m the Veteran’s caretaker"
+ *   }
+ * })
+ * ```
+ *
+ * @param {string | {
+ *   personTitle?: string,
+ *   relativeTitle?: string,
+ *   tense?: string,
+ *   labelHeaderLevel?: UISchemaOptions['ui:options']['labelHeaderLevel'],
+ *   customLabels?: object
+ * }} options - a string representing the person type, or an object with options
+ * @returns {UISchemaOptions}
+ */
+
+export const relationshipToVeteranUI = options => {
+  const { personTitle, relativeTitle, tense, labelHeaderLevel, customLabels } =
+    typeof options === 'object' ? options : { personTitle: options };
+
+  const person = personTitle ?? 'Veteran';
+
+  const isRelativeFn = typeof relativeTitle === 'function';
+  const relativeTitleVal = isRelativeFn
+    ? props => relativeTitle(props)
+    : relativeTitle;
+
+  const conjugatedVerbThirdPerson = tense && tense === 'past' ? 'was' : 'is';
+  const conjugatedVerbFirstPerson = tense && tense === 'past' ? 'I was' : 'I’m';
+
+  const relativeBeingVerb = `${`${
+    relativeTitleVal
+      ? `${relativeTitleVal} ${conjugatedVerbThirdPerson}`
+      : conjugatedVerbFirstPerson
+  }`}`;
+
+  const relativePossessive = `${`${
+    relativeTitleVal ? `${relativeTitleVal}’s` : 'your'
```

Comment:
These nested template strings combined with a ternary make readability sort of difficult. I think this is an instance where just sticking to plain old if/else statements to assign a value to `relativeBeingVerb` may be the way to go.